### ISSUE TITLE
Cyclic learning rates

### ISSUE THREAD

**Issue by JohnGiorgi** [BaderLab/Saber#56]
We should see if using the cyclic learning rate finder (paper: [here](https://arxiv.org/abs/1506.01186)) along with an adaptive learning rate optimizer (e.g., adam) improves on our current optimizer (nadam).

__Todo__

- [ ] Use the Cyclic LR Keras Callback to determine an optimal learning rate.
- [ ] Try this learning rate with a few different optimizers (starting with Adam). Does it beat our current optimization method?

__Resources__

- [Cyclic LR paper](https://arxiv.org/abs/1506.01186)
- [Keras Callback for Cyclic LRs](https://github.com/keras-team/keras-contrib/blob/master/keras_contrib/callbacks/cyclical_learning_rate.py)
